portal-impl/src/com/liferay/portal/service/impl/UserGroupLocalServiceImpl.java

```diff
@@ -133,6 +133,7 @@ public boolean addGroupUserGroup(long groupId, UserGroup userGroup) {
 
 		try {
 			reindexUsers(userGroup);
+			reindexUserGroup(userGroup);
```

Comment:
In line `106` there is another method `addGroupUserGroup` that also needs this change.